src/lib/util.ts

```diff
@@ -265,6 +270,10 @@ export function isTobActivity(data: any): data is TheatreOfBloodTaskOptions {
 	return 'wipedRoom' in data;
 }
 
+export function isNexActivity(data: any): data is NexTaskOptions {
+	return 'wipedKill' in data;
```

Comment:
This should definitely be a *bit* more robust, highly likely something else could have 'wipedKill'. e.g. check 2-3 properties? I will just commit this myself to save time.

Reply:
Yea I checked to make sure nothing else was already using it, but you're right in the future could definitely happen